hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DFSInputStream.java

```diff
@@ -181,6 +181,8 @@ private boolean isPeriodicRefreshEnabled() {
   private byte[] oneByteBuf; // used for 'int read()'
 
   protected void addToLocalDeadNodes(DatanodeInfo dnInfo) {
+    DFSClient.LOG.debug("Add {} to local dead nodes, previously was {}",
```

Comment:
1、Is it necessary to add log in Local dead nodes?
2、if add debug log in DFSInputStream#addToLocalDeadNodes, do you need add log in DFSInputStream#removeFromLocalDeadNodes?

Reply:
1. From the statistics of the production environment, the log output of local `deadnode` information is too few, so that it is troublesome to find the problem reason, now add it as `DEBUG`
2. The reason to use the `DEBUG` log level is to separate from the `INFO` log level of the detector, and the impact is small.
3. Remove logs can be added at the same time(I think both are OK). It was not added before considering that the local dead node removal method will almost never be called, unless the detector is turned on and then called, and there is already log output there.